Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2337,9 +2337,13 @@ void MediaPlayerPrivateGStreamer::updateStates()
                 GRefPtr<GstQuery> query = adoptGRef(gst_query_new_buffering(GST_FORMAT_PERCENT));
 
                 m_isBuffering = m_bufferingPercentage < 100;
-                if (gst_element_query(m_pipeline.get(), query.get())) {
+                if ((m_audioSink && gst_element_query(m_audioSink.get(), query.get()))
+                    || (m_videoSink && gst_element_query(m_videoSink.get(), query.get()))
+                    || gst_element_query(m_pipeline.get(), query.get()))
+                {
```

Comment:
Taking advantage of having to resubmit the patch because the "Reviewed by" line was absent, please fix the breakage of the [braces-blocks rule of the coding style](https://webkit.org/code-style-guidelines/#braces-blocks) and place the opening brace at the end of the previous line, instead of on its own line. Braces on their own lines are [reserved for method/function definitions](https://webkit.org/code-style-guidelines/#braces-function).

Thanks in advance!

Reply:
What is weird is that the style bot didn't catch this...